server/src/main/java/org/candlepin/auth/KeycloakOIDCFacade.java

```diff
@@ -210,10 +209,10 @@ public Response getResponse() {
 
     @Override
     public X509Certificate[] getCertificateChain() {
-        throw new IllegalStateException("Not supported yet");
+        throw new UnsupportedOperationException("Not supported yet");
     }
 
     public boolean isResponseFinished() {
-        return responseFinished;
+        return true;
```

Comment:
This is different now, but it doesn't address the problem cited: now we're always returning true, and we still have a useless variable in responseFinished that is never updated or (now) referenced.